packages/appstore/src/components/cfds-listing/index.tsx

```diff
@@ -191,6 +192,11 @@ const CFDsListing = observer(() => {
                                       existing_account?.landing_company_short
                                   )
                                 : null;
+                        //TODO replace status with open_order_position_status once key is available in BE response and in type TSocketResponseData<"mt5_login_list">
```

Comment:
Here "landing_company_short" should be replaced with open_order_position_status. isnt?

Reply:
yes!